core/src/test/scala/tamer/utils/FakeConsumer.scala

```diff
@@ -139,6 +155,10 @@ sealed class FakeConsumer[IK, IV](
   override def subscribeAnd(subscription: Subscription): SubscribedConsumer                                                             = ???
   override def subscription: Task[Set[String]]                                                                                          = ???
   override def metrics: Task[Map[MetricName, Metric]]                                                                                   = ???
+  override def partitionedAssignmentStream[R, K, V](
+      keyDeserializer: Deserializer[R, K],
+      valueDeserializer: Deserializer[R, V]
+  ): stream.Stream[Throwable, Chunk[(TopicPartition, ZStream[R, Throwable, CommittableRecord[K, V]])]] = ???
```

Comment:
```suggestion
  ): Stream[Throwable, Chunk[(TopicPartition, ZStream[R, Throwable, CommittableRecord[K, V]])]] = ???
```
to fix also in `partitionedStream` method signature